static/json/languages/en_US.json

```diff
@@ -495,6 +495,7 @@
       },
       "search_tags": {
         "swap": "bridge",
+        "bridge": "bridge",
```

Comment:
@DanielSinclair Do you have any suggestions on what search tags we can add here for Bridge shortcut ?

Reply:
@magiziz Added a few tweaks!